docs/DeveloperGuide.md

```diff
@@ -331,6 +337,54 @@ Extensions:
 - 1a. Tag already exists.
   - 1a1. FoodRem displays error warning to user.
     Use case resumes from step 1.
+  
+#### UC7: Rename a tag
+
+Use Case: UC7 - Rename a tag
+MSS:
+
+1. User chooses to view all tags.
+2. System shows all tags.
```

Comment:
Possibly edit this in the future to reference the use case for viewing all tags instead